src/lib/Scenes/Sale/Components/RegisterToBidButton.tsx

```diff
@@ -1,16 +1,16 @@
+import { ContextModule, ScreenOwnerType, tappedRegisterToBid, TappedRegisterToBidArgs } from "@artsy/cohesion"
```

Comment:
```suggestion
import { ContextModule, OwnerType, ScreenOwnerType, tappedRegisterToBid, TappedRegisterToBidArgs } from "@artsy/cohesion"
```

This suggestion goes along with my suggestion below.